lib/kbsecret/cli/kbsecret-cp

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "kbsecret"
+include KBSecret
+
+cmd = CLI.create do |c|
+  c.slop do |o|
+    o.banner = <<~HELP
+      Usage:
+        kbsecret cp <src-session> <target-session> [options] <record [record ...]>
+    HELP
+    o.bool "-m", "--move", "delete the record after copying"
+  end
+
+  c.dreck do
+    string :session
```

Comment:
Let's make these `src_sess` and `dst_sess` (or something equivalent, your call) for consistency.